CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Some of the gathered credentials no longer appear in database plaintext. #1454
 - Encryptor breaking with UTF-8 characters. (Passwords in different languages can be submitted in
   the config successfully now.) #1490
-
+- Malfunctioning Modify shell startup files PBA on Docker. #1507
```

Comment:
Changelog entries should be meaningful to the user of the changed component. "Malfunction" is a little bit broad, and the issue isn't specifically docker related.

> Unhandled error when "modify shell startup files PBA" is unable to find regular users. #1507